angular/projects/admin-nrpti/src/app/import/import.component.html

```diff
@@ -1,22 +1,47 @@
 <main class="container-fluid-padding">
   <section class="mb-3">
     <h4>Import from Source System</h4>
-    <div class="mb-4">
-      <div class="label-pair">
-        <label>Import from EPIC</label>
-        <div>
-          <button type="button" [disabled]="showAlert" class="btn btn-primary" (click)="startJob('epic')">
-            Start Job
-          </button>
-        </div>
+
+    <div class="label-pair mb-3">
+      <label>Import from EPIC</label>
+      <div>
+        <button
+          type="button"
+          [disabled]="showAlert"
+          class="btn btn-primary"
+          (click)="startJob('epic')"
+          title="Start EPIC import job"
+        >
+          Start Job
+        </button>
+      </div>
+    </div>
+    <div class="label-pair mb-3">
+      <label>Import from NRIS</label>
+      <div>
+        <button
+          type="button"
+          [disabled]="showAlert"
+          class="btn btn-primary"
+          (click)="startJob('nris-epd')"
+          title="Start NRIS import job"
+        >
+          Start Job
+        </button>
       </div>
-      <div class="label-pair">
-        <label>Import from NRIS</label>
-        <div>
-          <button type="button" [disabled]="showAlert" class="btn btn-primary" (click)="startJob('nris-epd')">
-            Start Job
-          </button>
-        </div>
+    </div>
+    <div class="label-pair mb-3">
+      <label>Import from CORE</label>
+      <div>
+        <button
+          type="button"
+          [disabled]="showAlert"
+          class="btn btn-primary"
+          (click)="startJob('core')"
+          title="Start CORE import job"
+        >
+          Start Job
+        </button>
```

Comment:
This isn't part of the JIRA task, and also I believe we might be moving away from having those buttons for core/epic/nris in the UI in the first place.  Instead possibly moving to adjusting the schedule, rather than letting people spam the button and causing DOS of the respective service.  We currently have no controls in place for this.

As a side-note, I also don't believe showAlert should disable anything other than the button in question since logically they weren't tied together in any way.

Reply:
Updated in https://github.com/bcgov/NRPTI/pull/419/commits/fcc824f7dc72e7277402d57afa1680f13e10ee0a